cli/parser/parser.go

```diff
@@ -116,3 +118,52 @@ func GetFlagValue(options []*BazelOption, phase, config, flagName, commandLineOv
 	}
 	return ""
 }
+
+func GetArgsFromRCFiles(commandLineArgs []string) []string {
+	rcFiles := make([]string, 0)
+	if arg.Get(commandLineArgs, "system_rc") != "true" {
+		rcFiles = append(rcFiles, "/etc/bazel.bazelrc")
+		rcFiles = append(rcFiles, "%ProgramData%\bazel.bazelrc")
+	}
+	if arg.Get(commandLineArgs, "workspace_rc") != "true" {
+		rcFiles = append(rcFiles, ".bazelrc")
+	}
+	if arg.Get(commandLineArgs, "home_rc") != "true" {
+		usr, err := user.Current()
+		if err == nil {
+			rcFiles = append(rcFiles, filepath.Join(usr.HomeDir, ".bazelrc"))
+		}
+	}
+	if b := arg.Get(commandLineArgs, "bazelrc"); b != "" {
```

Comment:
`--bazelrc` can be specified multiple times -- fix or add TODO?

Reply:
This is an issue with the current logic as well, so I've just added a TODO for now.